libcontainer/container_linux.go

```diff
@@ -376,6 +383,14 @@ func (c *linuxContainer) Checkpoint(criuOpts *CriuOpts) error {
 		}
 	}
 
+	// append optional manage cgroups mode
+	if criuOpts.ManageCgroupsMode != 0 {
+		if err := c.checkCriuVersion("1.6"); err != nil {
```

Comment:
@avagin @xemul Thanks for the instructions. So the pseudo code would be
output := "criu --version"
if output containers "GitID"
    // use the line GitID: 1.6.2-smth or 1.6-smth
    // increment the number by one
else
    // use the line Version: 1.6 as in the exisitng code

Correct?